sdk/java-sdk-protobuf/src/main/scala/kalix/javasdk/impl/HttpResponseImpl.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk.impl
+
+import kalix.javasdk.HttpResponse
+import kalix.javasdk.StatusCode
+
+private[kalix] class NoContent(statusCode: StatusCode.Success) extends HttpResponse {
```

Comment:
Actually this type can be removed at this point.

Reply:
We should keep it, but have the status code fixed to 204

Reply:
Maybe we should suffix all this types with `Response`, eg: `NoContentResponse`, `JsonContentResponse`. WDYT?

Reply:
done.